packages/config/config/devices/0x010f/fgd212_3.5.json

```
@@ -846,75 +805,80 @@
 			]
 		},
 		"50": {
-			"label": "Active power reports",
-			"description": "Pwr level change resulting in power report sent",
+			"label": "Active power report",
+			"description": "The power level change as a percentage from the previous report that will result in a new power report being sent",
```

This is quite verbose and could probably be shortened, for example:
```suggestion
			"label": "Power Report: Relative Change Threshold",
```

---

Would `Active Power Report: Relative Change Since Previous Report` be ok, or still to long? I wanted to keep something saying that the previous report was the reference since that might not be obvious.

Could move that to the description though and use your suggestion as the label.

---

I'd prefer that to be in the description. If the label looks like it could be a sentence, it is too long IMO.